UI/src/view/linear_asset/linearAssetLabel.js

```diff
@@ -313,6 +322,12 @@
       var getTypeId = function (asset) {
         return asset.typeId;
       };
+      
+      me.isSuggested = function(asset) {
+        return !!parseInt(_.head(getValues(asset)).isSuggested);
```

Comment:
Can this !!parseInt() be replaced with !isNaN() to be more recognizable about what you are doing here?